paas/install_cli.go

```diff
@@ -0,0 +1,41 @@
+package paas
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"runtime"
+
+	"github.com/fuseml/fuseml/cli/helpers"
+	"github.com/fuseml/fuseml/cli/paas/ui"
+	"github.com/pkg/errors"
+)
+
+const (
+	coreClientDownloadURL = "https://github.com/fuseml/fuseml-core/releases/latest/download"
+	coreClientName        = "fuseml"
+)
+
+// download platform specific fuseml-core client to current directory
+func downloadClient(ui *ui.UI) error {
```

Comment:
I'd rename it to `downloadCoreClient` - now that it is not in the context of `core` deployment, just "client" does not say much

Reply:
That's exactly what I was trying to correct with the rename from `fuseml_core-cli` to `fuseml`: there is no FuseML "core" CLI, there is just the FuseML CLI and the FuseML installer. The core is just an architectural artifact that the end user doesn't care about.

Best I can do is `downloadFuseMLCLI` or ``downloadFuseMLClient` instead of `downloadClient`, but "core" has no place here.

Reply:
Well, I do not think "core" has no place there. This is a code, no user facing documentation. And from the code POV, it really is the client that is part of "core" service which is another package, another repository.

Just like there's "core" deployment.

Reply:
Your points would make more sense if "client" were an application independent on the "core" package, which it is not.